.github/workflows/preliminary_checks.yml

```diff
@@ -70,3 +75,23 @@ jobs:
             echo "\`\`\`" >> $GITHUB_STEP_SUMMARY
             exit 1
           fi
+
+  title-check:
+    name: CC PR Titel
+    runs-on: ubuntu-latest
+    steps:
+      - shell: python
+        env:
+          title: "${{ github.event.pull_request.title }}"
+        run: |
+          import re 
+          import os
+          title = os.environ["title"]
+          title_re = r"^(feat|fix|build|test|docs|refactor|misc)(\(.+\))?: (.+)"
```

Comment:
it would be extra nice if we could ensure people start the description with a Capital letter and do not end with a period "."

Reply:
Added